build/images/ovs/Dockerfile.windows

```diff
@@ -0,0 +1,47 @@
+# Copyright 2023 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG NANOSERVER_VERSION
+
+FROM mcr.microsoft.com/windows/servercore:${NANOSERVER_VERSION} as windows-ovs
+
+ARG WIN_OVS_VERSION
+
+# Switch to a privileged user
+USER ContainerAdministrator
+
+# Sets the shell to be PowerShell 
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+
+ENV WIN_OVS_VERSION ${WIN_OVS_VERSION}
+
+# Install VC for ovs
+RUN curl.exe -Lo C:\vc_redist.x64.exe https://aka.ms/vs/17/release/vc_redist.x64.exe; \
+    powershell -Command  "Start-Process -FilePath C:\vc_redist.x64.exe -ArgumentList ‘/install’,’/quiet’,’/norestart’ -Verb RunAs -Wait"; \
+    rm C:\vc_redist.x64.exe
+
+# Download and extract OVS
+RUN curl.exe -fLo C:\ovs-${env:WIN_OVS_VERSION}-antrea.0-win64.zip https://downloads.antrea.io/ovs/ovs-${env:WIN_OVS_VERSION}-antrea.0-win64.zip; \
+    powershell -Command Expand-Archive C:\ovs-${env:WIN_OVS_VERSION}-antrea.0-win64.zip -DestinationPath C:\  ; \
+    rm C:\ovs-${env:WIN_OVS_VERSION}-antrea.0-win64.zip
+
+# Download, extract OpenSSL and copy libraries to ovs
+RUN curl.exe -Lo C:\openssl-1.0.2u-x64_86-win64.zip https://indy.fulgan.com/SSL/openssl-1.0.2u-x64_86-win64.zip; \
+    powershell -Command Expand-Archive C:\openssl-1.0.2u-x64_86-win64.zip -DestinationPath C:\  ;  \
+    cp C:/libeay32.dll C:/openvswitch/usr/sbin/ ; \
+    cp C:/ssleay32.dll C:/openvswitch/usr/sbin/ ; \
+    mv C:/libeay32.dll C:/openvswitch/usr/bin/ ; \
+    mv C:/ssleay32.dll C:/openvswitch/usr/bin/ ; \
+    mkdir C:/openvswitch/var/run/openvswitch ; \
+    Remove-Item -Path 'openssl-1.0.2u-x64_86-win64.zip', 'HashInfo.txt', 'License.txt', 'openssl.exe', 'ReadMe.txt', 'OpenSSL License.txt'
```

Comment:
If my understanding is correct, we will run OVS inside a container based on this image and do not copy the OVS files to any other images. Then I would suggest to directly create the OVS db.conf in this image and set the ovs_version in OVSDB. In this way, we don't need to perform the two steps in antrea-agent's init-container steps, e.g., append these steps,

```
# Prepare OVS db.conf and set OVS version
RUN c:/openvswitch/usr/bin/ovsdb-tool.exe create c:/openvswitch/etc/openvswitch/conf.db c:/openvswitch/usr/share/openvswitch/vswitch.ovsschema ; \
    c:/openvswitch/usr/sbin/ovsdb-server.exe c:/openvswitch/etc/openvswitch/conf.db --remote=punix:db.sock --detach ; \
    $OVS_VERSION=$(Get-Item c:/openvswitch/driver/OVSExt.sys).VersionInfo.ProductVersion ; \
    c:/openvswitch/usr/bin/ovs-vsctl.exe --no-wait set Open_vSwitch . ovs_version=$OVS_VERSION ; \
    Get-Process ovsdb-server | Stop-Process -Force
```

Reply:
BTW, it is better to add the binary path "c:/openvswitch/usr/bin" into "$env:PATH" with computer level, then we can run ovs-vsctl/ovs-ofctl/ovs-dpctl commands in antrea-ovs container in the rumtime. e.g.,

```
RUN cmd /S /C setx /m PATH "C:/openvswitch/usr/bin;%PATH%" 
```